odoo/service/server.py

```diff
@@ -53,7 +53,8 @@
     setproctitle = lambda x: None
 
 import odoo
-from odoo.modules import get_modules
+from odoo.modules import get_modules, load_addons_manifest
+from odoo.modules.module import addons_manifest # for backward compatibility
```

Comment:
backwards compatibility with what?

Reply:
There was a lot of python code that called http and now has to do module. This is for external code (partner, or our instance)

Reply:
ho, it's the wrong file... error when I moved some code